docs/guide/ingress/annotations.md

```diff
@@ -816,6 +817,14 @@ Custom attributes to LoadBalancers and TargetGroups can be controlled with follo
                     alb.ingress.kubernetes.io/target-group-attributes: load_balancing.algorithm.type=least_outstanding_requests
                     ```
 
+- <a name="target-group-prefix">`alb.ingress.kubernetes.io/target-group-prefix`</a> specifies the name prefix given to Target Groups. The prefix is truncated to 22 characters.
+
+    !!!example
+        - set the target group prefix to "my-service-", the full name will look like "my-service-8f5fe20735"
+            ```
+            alb.ingress.kubernetes.io/target-group-prefix: my-service-
```

Comment:
I think it would make sense to imply the trailing dash. So instead of `my-service-` it would just be `my-service` and the dash would be added in the concatenation statement